package.json

```diff
@@ -38,9 +38,9 @@
   ],
   "devDependencies": {
     "@types/node": "^8.0.4",
-    "@types/request": "^0.0.45",
-    "ava": "^0.22.0",
-    "coveralls": "^2.11.2",
+    "@types/request": "^2.0.8",
```

Comment:
think we could omit the types here, IDEs can do this now.

Reply:
Doesn't the TS compiler check it if this is included? Is it only for IDE sugar?

Reply:
(as in, throws compilation errors)